src/domain/community/organisation/organisation.service.ts

```diff
@@ -44,7 +46,10 @@ export class OrganisationService {
     return organisation;
   }
 
-  async initialiseMembers(organisation: IOrganisation): Promise<IOrganisation> {
+  async initialiseMembers(
+    organisation: IOrganisation,
```

Comment:
Shouldn't CreateOrganisationInput as DTO contain all data needed to create an organisation? What does the first parameter bring?

Reply:
Actually realised we do not need this split with initialiseMembers any more...merged into one function